docs/sphinx/examples/python/qaoa_maxcut.py

```diff
@@ -26,28 +28,26 @@
 parameter_count: int = 2 * layer_count
 
 
-def kernel_qaoa() -> cudaq.Kernel:
+@cudaq.kernel
+def kernel_qaoa(qubit_count: int, layer_count: int, thetas: List):
```

Comment:
The data type needs to be fully defined - I believe this sample should not longer work as is:
```suggestion
def kernel_qaoa(qubit_count: int, layer_count: int, thetas: List<float>):
```